pyfr/plugins/tavg.py

```diff
@@ -220,6 +226,46 @@ def _acc_avg_var(self, intg, currex):
             if self.tstart_acc != self.prevt:
                 v +=  (Wmp1mpn / Wp * (a - W1mpn * ppc)**2)
 
+    @staticmethod
+    def merge_cli(args):
+        a_out = h5py.File(args.output, 'w')
+        start_time = np.empty((0, 1))
+        end_time = np.empty((0, 1))
+
+        for i, filename in enumerate(args.solns):
```

Comment:
Also need to handle functional averages (they need re-evaluating based off of the new data) and also code to handle the variance calculations (both summary and full outputs).  @rishit2307 can help you here.